firestore-bigquery-export/scripts/gen-schema-view/src/index.ts

```diff
@@ -112,17 +141,71 @@ async function run(): Promise<number> {
   return 0;
 };
 
-function readSchemas(directory: string): { [schemaName: string]: FirestoreSchema } {
+function readSchemas(globs: string[]): { [schemaName: string]: FirestoreSchema } {
+  let schemas = {};
+  let expanded = expandGlobs(globs);
+  for (var i = 0; i < expanded.length; i++) {
+    let dirent = resolveFilePath(expanded[i]);
+    let stats = lstatSync(dirent);
+    if (stats.isDirectory()) {
+      let directorySchemas = readSchemasFromDirectory(dirent);
+      for (let schemaName in directorySchemas) {
+        if (schemas.hasOwnProperty(schemaName)) {
+          warnDuplicateSchemaName(schemaName);
+        }
+        schemas[schemaName] = directorySchemas[schemaName];
+      }
+    } else {
+      let schemaName = filePathToSchemaName(dirent);
+      if (schemas.hasOwnProperty(schemaName)) {
+        warnDuplicateSchemaName(schemaName);
+      }
+      schemas[schemaName] = readSchemaFromFile(dirent);
+    }
+  }
+  return schemas;
+}
+
+function warnDuplicateSchemaName(schemaName: string) {
+  console.log(`Found multiple schema files named ${schemaName}! Only the latest one will be created!`);
```

Comment:
```suggestion
  console.log(`Found multiple schema files named ${schemaName}! Only the last one will be created!`);
```
I don't think there is a time order associated with schema files?